### ISSUE TITLE
Is tf.nn.depthwise_convolution is the correct way to implement resnext bottleneck_c?

### ISSUE THREAD

**Issue by lsy643** [wenxinxu/ResNeXt-in-tensorflow#2]
The tf.nn.depthwise_convolution does convolution for every channel of the feature map, which I think does not match the paper.

**Comment by wenxinxu** [comment]
You are right! depthwise_conv seems not to be the correct way for group conv. The split->concat method should work though.

 I have modified the readme.md. Thanks for pointing out!
